build/README-DB2.md

```diff
@@ -11,15 +11,16 @@ Running fhir server integration tests using docker db2.
 
 ## Run
 
-From the root of the project (aka the "WORKSPACE"):
-1. Set up and start the containers
+From the root of the project (aka the "WORKSPACE"):  
+1. Set up and start the containers  
     ```sh
     build/pre-integration-test-docker.sh
-    ```
-2. Run the tests
+    ```  
+    Note: If you are testing NATS notifications, invoke fhir-server-test/src/test/nodejs/nats-subscriber.js via `node nats-subscriber`.  
```

Comment:
lets give the full path to the executable and add a note about building it```suggestion
    Note: If you are testing NATS notifications, invoke fhir-server-test/src/test/nodejs/nats-subscriber.js via `node fhir-server-test/src/main/nodejs/nats-subscriber.js`.  If this is your first time, install the dependencies first by installing [Node.j](https://nodejs.org/en/download) (if not already installed) and running `(cd fhir-server-test/src/main/nodejs/ && npm install)`
```

Reply:
We have to add a package.json and should add package-lock.json.  Also, the current location of nats-subscriber.js is: fhir-server-test/src/test/nodejs/nats-subscriber.js

Reply:
no need to use package-lock.json here, so just added package.json.

Reply:
great, i thought i had copied that over when i moved to fhir-server-test/src/main/nodejs but i must have dropped it by accident